src/twisted/internet/test/test_base.py

```diff
@@ -183,6 +183,7 @@ def setUp(self):
         Create two L{DelayedCall} instanced scheduled to run at different
```

Comment:
If you don't mind changing `instanced` to `instances` here, that'd be nice.  :-)